crates/sui-cluster-test/src/faucet.rs

```diff
@@ -29,9 +29,7 @@ impl FaucetClientFactory {
                     .local_faucet_key()
                     .expect("Expect local faucet key for local cluster")
                     .copy();
-                let wallet_context = new_wallet_context_from_cluster(cluster, key)
-                    .instrument(info_span!("init_wallet_context_for_faucet"))
```

Comment:
actually, can we add this instrument and info_span back?

Reply:
the `new_wallet_context_from_cluster` was updated to be synchronous, does it need to do instrument tracing?
i look document, it normal use in async func..  Just curious.